lib/application/pages/execution/execution_terminal.dart

```diff
@@ -158,6 +159,7 @@ class ExecutionTerminal extends HookWidget {
         child: Opacity(
           opacity: curvedController.value,
           child: _TerminalActions(
+            key: _terminalActionsKey,
```

Comment:
shouldn't this be a constant `ValueKey` if you want it to not change between multiple builds? Also, why exactly adding a key here?